### ISSUE TITLE
Don't translate Scripture references in non-verse text

### ISSUE THREAD

**Issue by johnml1135** [sillsdev/serval#573]
There are often Scripture references at the beginning or end of non-verse text.  Currently we are trying to translate these and making a mess of things.  We should be able to do better.  Here are some options:

Best proposal:
* Strip out all references when making training/pretranslating data
* Re-insert them when putting into USFM

Complications:
* How do we know something is a scripture reference?  Can we truly make a regex to capture all references?  What about multiple languages?
  * This one should work for english: https://stackoverflow.com/questions/66818564/regex-for-bible-references
* Re-insertions would require reworking the USFM updater - which may add some complication (but not too much because there will be no ranges do deal with in non-verse scripture text).

**Comment by ddaspit** [comment]
We should just be more careful about what markers we translate and what markers we don't translate.

**Comment by johnml1135** [comment]
Yes - we could try some funny things with regex's - but I think just the "translate these tags" and "don't translate these tags" is the best way to get to the 95%.

**Comment by johnml1135** [comment]
This parent issue no longer has any further tasks than the child has.  We can cancel this (and just do the child issue).
